src/Shared/src/CommandLineUtils/Microsoft.Extensions.CommandLineUtils.Sources.projitems

```diff
@@ -1,14 +0,0 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
-<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
```

Comment:
Can you also either restore this or update https://github.com/aspnet/Extensions/blob/rybrande/Arcade/eng/tools/BaselineGenerator/BaselineGenerator.csproj#L13? I'm assuming that tool doesn't build anymore as a result of deleting this file.